### ISSUE TITLE
Main window fails to open when no folders exist

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#62]
```
I have two computers. One I already had AutoKey installed and this one I am 
trying to add it.

I've got it added now to the latest version but it wasn't the most simple 
process.

I tried to install AK 0.71.0 after adding the PPA to my sources but it gave me 
an error saying that one of the dependencies was untrusted.

I forgot to make a screenshot of the error but I think the dependency that it 
said was untrusted was "python-xlib" (I know it was one of the pythons though, 
for some reason I remember a w you in it like - python-wlib sorry for not being 
much help on that part.)

I got 0.71.0 installed though by first installing it through the Ubuntu 
Software Center (0.67.1) and then activating the PPA and using the Ubuntu 
Update Manager to update it to .0.71.0

It could be just an issue with launchpad but I thought you should know.
```

Original issue reported on code.google.com by `vis...@gmail.com` on 1 Aug 2010 at 9:02

**Comment by GoogleCodeExporter** [comment]
```
Misconfiguration on client system or launchpad
```

Original comment by `cdekter` on 2 Aug 2010 at 12:39
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
so any ideas on a solution?
```

Original comment by `vis...@gmail.com` on 2 Aug 2010 at 12:48

**Comment by GoogleCodeExporter** [comment]
```
Did you add the PPA key to your software sources?
```

Original comment by `cdekter` on 2 Aug 2010 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
I added "ppa:cdekter/ppa"
```

Original comment by `vis...@gmail.com` on 2 Aug 2010 at 12:55

**Comment by GoogleCodeExporter** [comment]
```
In that case I'm not sure - it should work. I haven't seen this problem before. 
Either way, it should be possible to simply bypass the untrusted message as 
it's only a warning. And without knowing exactly the name of the dependency, I 
can't help.
```

Original comment by `cdekter` on 2 Aug 2010 at 1:02

**Comment by GoogleCodeExporter** [comment]
```
It's actually not just a warning, unfortunately it halts the install, 
unfortunately there isn't a way to continue...once it happens, you click "OK" 
and it stops the install.

I'll remove the program and try again to find what dependency it is.
```

Original comment by `vis...@gmail.com` on 2 Aug 2010 at 1:06

**Comment by GoogleCodeExporter** [comment]
```
just tested it...maybe it was just a fluke or something. I removed AK 
completely and then installed straight from the PPA and it worked so not sure 
why it was doing that earlier.
```

Original comment by `vis...@gmail.com` on 2 Aug 2010 at 2:22

**Comment by GoogleCodeExporter** [comment]
```
ok I removed it after the last install because it wouldn't load the GUI for 
some reason and I then tried to reinstall again and the error happened again - 
http://i26.tinypic.com/cl3wh.jpg
```

Original comment by `vis...@gmail.com` on 2 Aug 2010 at 2:37

**Comment by GoogleCodeExporter** [comment]
```
This can only happen if you haven't got the GPG key for the PPA properly 
installed. Try adding it manually - the instructions are on the PPA page on 
launchpad
```

Original comment by `cdekter` on 2 Aug 2010 at 2:39

**Comment by GoogleCodeExporter** [comment]
```
ok I did it manually through terminal and it installed just fine but now the 
GUI wont load for me. Could it be because of my laptop being a 64bit?
```

Original comment by `vis...@gmail.com` on 2 Aug 2010 at 2:58

**Comment by GoogleCodeExporter** [comment]
```
No it can't have anything to do with that. Try running autokey-gtk from the 
command line and post the output
```

Original comment by `cdekter` on 2 Aug 2010 at 3:04

**Comment by GoogleCodeExporter** [comment]
```
sorry about the delay on the debug info...

Here is the debug info for when I launch the app:
http://tinypaste.com/3e113e

Here is the debug for when I try to open the Main Window by clicking on the 
tray icon:
http://tinypaste.com/006063

Here is the debug for when right clicking the tray icon and choosing "Show Main 
Window":
http://tinypaste.com/0f8e0
```

Original comment by `vis...@gmail.com` on 6 Aug 2010 at 4:13

**Comment by GoogleCodeExporter** [comment]
```
Try removing the ~/.config/autokey folder (or rename it if you want to back up 
your existing autokey settings) and then running autokey again
```

Original comment by `cdekter` on 6 Aug 2010 at 4:21

**Comment by GoogleCodeExporter** [comment]
```
that fixed the GUI issue...thanks for the assistance.
```

Original comment by `vis...@gmail.com` on 6 Aug 2010 at 5:16

**Comment by GoogleCodeExporter** [comment]
```
ok I figured out what caused the GUI not loading issue.

I had deleted all of the folders in order to import my items from my other 
computer but when I deleted them all and closed the GUI it wouldn't load GUI 
again until I did what you suggested.

So I tested it multiple times and turns out if there isn't at least one folder 
the GUI won't load.
```

Original comment by `vis...@gmail.com` on 6 Aug 2010 at 5:45

**Comment by GoogleCodeExporter** [comment]
Original comment by `cdekter` on 6 Aug 2010 at 5:52
- Changed title: **Main window fails to open when no folders exist**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed for GTK UI in v0.71.1. KDE still TODO
```

Original comment by `cdekter` on 22 Oct 2010 at 2:54
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
KDE UI is now deprecated so this is considered done
```

Original comment by `cdekter` on 12 Sep 2011 at 2:42
- Changed state: **Fixed**
